osquery/tables/networking/aws_metadata.h

```diff
@@ -0,0 +1,113 @@
+/*
```

Comment:
the initial idea was to develop next another for the host network interfaces and its associated data: subnet, vpc id etc.. hence i used a header file to be used by that table implementation. The reality is that 99.9999% of the people will only use 1 network interface on those hosts and i see little added value. I am adding a few fields to the table to get the subnet data directly into the main table.